app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -241,8 +243,32 @@ public void onResume() {
             _hlEditor.setText(CoolExperimentalStuff.convertEpubToText(_document.getFile(), getString(R.string.page)));
         }
 
-        // Set initial wrap state
         initDocState();
+
+        // Run once in first onResume if visible. Should always be visible here
```

Comment:
onResume is a better place for this logic as the UI is fully setup here. Flicker was being caused by conflicts between view generation and setting selection etc